core/runtime/src/main/java/io/quarkus/runtime/configuration/ConverterSupport.java

```diff
@@ -19,6 +19,7 @@
  * This small utility class is a tool which helps populating SmallRye {@link ConfigBuilder} with
  * {@link Converter} implementations loaded from {@link ServiceLoader}.
  */
+// todo: delete
```

Comment:
Should this be addressed?

Reply:
Yes, this can be done now.  I'll add a commit.